core/src/types/capability.rs

```diff
@@ -88,6 +88,8 @@ pub struct Capability {
 
     /// If operator supports append natively, it will be true.
     pub append: bool,
+    /// If operator supports append to a file not created by append natively, it will be true.
+    pub append_normal: bool,
```

Comment:
No, we should not have this.

Reply:
Some services do not distinguish between file types.

Reply:
You are right, but we don't need to support.

Reply:
This is not to support. Just a capability to represent and enable test. :thinking:

Reply:
> This is not to support. Just a capability to represent and enable test. thinking

No, there are two problems here:

- Capability is our public API. It cannot be added solely for testing purposes.
- Behavior test is our promise. Why should we need to add such *unsupported* things?

Reply:
This is a promise, isn't it? 🤨

Only having this capability means that this service can append to normal files.

Reply:
No, I will reject this PR.